audits.md

```diff
@@ -0,0 +1,4 @@
+# Audits
+
+- [The first audit by Peckshield](https://github.com/peckshield/publications/blob/master/audit_reports/PeckShield-Audit-Report-OlympusDAO-v1.0.pdf)
+  happened at the 9th of April 2021 for all deployed smart contracts at the time.
```

Comment:
```suggestion
[The first audit by PeckShield](https://github.com/peckshield/publications/blob/master/audit_reports/PeckShield-Audit-Report-OlympusDAO-v1.0.pdf)
  was completed on the 9th of April 2021. It covered all deployed smart contracts at the time.
```

Reply:
Not sure why you would add a bullet list, so I removed it

Reply:
No biggie. Just thought the second audit is underway and I think it will not be the last one.